app/main.py

```diff
@@ -1 +1,10 @@
-# write your code here
+def copy_file(command: str) -> None:
```

Comment:
It's not clear what the `command` argument represents in this function. It would be better to rename it to something more descriptive, like `copy_command`.